x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -292,6 +292,8 @@ const getSavedObjectTypes = (
               properties: {
                 synthetic_source: { type: 'boolean' },
                 tsdb: { type: 'boolean' },
+                doc_value_only_numeric: { type: 'boolean' },
+                doc_value_only_other: { type: 'boolean' },
```

Comment:
We're trying to limit mapping additions to what's strictly necessary. Sometimes, developers aren't aware that defining your field in mappings is only required if you need to query against them. Rest of the time, using `dynamic: false` is sufficient. 

So I need to ask: are these new properties used for querying? Is there any value defining them in the mappings?

Reply:
I think we don't need to query on these fields. @joshdover @ruflin Could you confirm?

Reply:
I can't think of any reason we'd be querying on these right now in the application.

Reply:
++. If we ever have to query on it, we could use a runtime as the total number of these docs will be reasonable small.